worlds/tunic/__init__.py

```diff
@@ -125,6 +127,7 @@ def generate_early(self) -> None:
                 self.options.lanternless.value = passthrough["lanternless"]
                 self.options.maskless.value = passthrough["maskless"]
                 self.options.hexagon_quest.value = passthrough["hexagon_quest"]
+                self.options.hexagon_quest_ability_type.value = passthrough["hexagon_quest_ability_type"]
```

Comment:
```suggestion
                self.options.hexagon_quest_ability_type.value = passthrough.get("hexagon_quest_ability_type", 0)
```
I wanna start following this pattern going forward, mostly because this means UT won't fail if you have an apworld with this option and it was genned with one without the option.